opus/application/apps/help/api_guide.md

```diff
@@ -836,7 +852,7 @@ Examples:
 }    
 %ENDCODE%
 
-* Retrieve raw images only for a Galileo SSI observation in JSON format.
+* Retrieve raw images ("Current" version) only for a Galileo SSI observation in JSON format.
```

Comment:
This is pointing out that we are losing functionality. There is no longer a way to ask for "all" versions of a product type. It used to be when you specified a product type, you got all the versions, but now you only get the "current" version. Perhaps it would be worth adding an "@all" feature to the `files` API?